nfpm.go

```diff
@@ -230,3 +233,43 @@ func WithDefaults(info *Info) *Info {
 
 	return info
 }
+
+// CopiedFile describes the source and destination of one file to copy into a
```

Comment:
the name is a bit misleading... its not a copied file, is a file to be copied... 🤔

Reply:
Good point. How about "FileToCopy" (and "FilesToCopy" for the method)?